edge/pkg/devicetwin/dttype/types_helper_test.go

```diff
@@ -599,33 +599,42 @@ func TestBuildDeviceTwinResult(t *testing.T) {
 
 // TestBuildErrorResult is function to test BuildErrorResult().
 func TestBuildErrorResult(t *testing.T) {
-	result := Result{BaseMessage: BaseMessage{Timestamp: time.Now().UnixNano() / 1e6,
```

Comment:
any reason why the timestamp is now removed ?

Reply:
That field was not used previously. Spotted it and just removed.

Reply:
Timestamp is assigned from the source code function. So once we assign a timestamp from test case , there is very less chance that the same may get assigned in source code as well which causes test case to fail randomly. So removed the timestamp check and checking only other fields in the struct.